routes/web.php

```diff
@@ -19,15 +20,27 @@
 Route::middleware('auth')->group(function () {
     Route::get('/', function () {
```

Comment:
Please delete this route if not use

Reply:
Hello, sir. May I ask what route?

Reply:
Sorry just change
Route::get('/dashboard', function () {

Reply:
Noted, sir.